library/build.gradle

```diff
@@ -52,6 +52,8 @@ dependencies {
     implementation 'com.google.android.gms:play-services-location:17.0.0'
 
     implementation 'com.gu.android:toolargetool:0.3.0'
+
```

Comment:
With these changes, is it possible to remove the dependency with com.rengwuxian.materialedittext?

Reply:
No, not yet. There are Widgets that still have the dependency with com.rengwuxian.materialedittext